pandas/io/sql.py

```diff
@@ -632,36 +632,9 @@ def read_sql(
     >>> pd.read_sql('test_data', 'postgres:///db_name')  # doctest:+SKIP
 
     Apply date parsing to columns through the ``parse_dates`` argument
-
-    >>> pd.read_sql('SELECT int_column, date_column FROM test_data',
```

Comment:
I removed these examples because they trigger UserWarnings that the format could not be inferred. There is still an example of `parse_dates` at the bottom that I think fits the spirit of showing what the keyword does